Prashant Kawatkar/Assignments/cypress/integration/assignments/ProceedToCheckout.js

```diff
@@ -0,0 +1,25 @@
+describe('My Second Test Suite', function() 
+{
+ 
+it('My FirstTest case',function() {
+ 
+ 
+cy.visit("https://rahulshettyacademy.com/seleniumPractise/#/")
+cy.get('.search-keyword').type('ca')
+cy.wait(2000)
```

Comment:
Instead of wait. Handle in Cypress.promise.

Reply:
Going through the course content, will apply the same

Reply:
Please apply promise